=========================================================================================================================

# How to determine the age of the Universe in a Jupyter Notebook

=========================================================================================================================

## 10 Methods 

=========================================================================================================================

=========================================================================================================================

# Universe Horizon Radius calculus in 3 minutes (method 1)
$$R_U=\frac{2.\hbar^2}{G.m_e.m_p.m_n}$$

In [1]:
from scipy import constants

In [2]:
R_U=2*constants.hbar**2/(constants.G*constants.m_e*constants.m_p*constants.m_n)
print(R_U)

1.3058515261319119e+26


## One obtains 13.802861522337478 Gly with method 1

In [3]:
R_U/(constants.light_year*10**9)

13.802861522337478

=========================================================================================================================

# Nambu Universe Radius calculus in 3 minutes (method 2)
$$R_N=\frac{2.\hbar^2}{G.(m_N)^3}$$

### https://en.wikipedia.org/wiki/Yoichiro_Nambu

With Nambu mass $$m_N=m_e.a$$ and $$a=1/\alpha$$

In [5]:
a=(1/constants.fine_structure)
print(a)

137.0359990836958


In [6]:
R_N=2*constants.hbar**2/(constants.G*((a)*constants.m_e)**3)
print(R_N)

1.7131902573578377e+26


## One obtains 18.108435308700802 Gly with method 2

In [7]:
R_N/(constants.light_year*10**9)

18.108435308700802

$$R_N'=ƛ_e(2R/R_N)^{210}$$ and $$R=\frac{2.\hbar^2}{G.m_e.m_p.m_H}$$

In [41]:
constants.physical_constants[]

SyntaxError: invalid syntax (<ipython-input-41-bd479024f253>, line 1)

In [ ]:
constants.

In [ ]:
m_H=constants.m_e

In [ ]:
R=2*constants.hbar**2/(constants.G*constants.m_e*constants.m_p*constants.m_H)
print(R)

In [7]:
R_Nprime=(constants.hbar/(constants.m_e*constants.c))*(2*R/R_N)**210

NameError: name 'R' is not defined

=========================================================================================================================

# Eddington Universe Radius calculus in 3 minutes (method 3)
$$R_{Edd}=\frac{2.\hbar^2}{G.m_e.m_p^2}$$

### https://en.wikipedia.org/wiki/Eddington_number

In [8]:
R_Edd=2*constants.hbar**2/(constants.G*constants.m_e*constants.m_p**2)
print(R_Edd)

1.3076515370853231e+26


## One obtains 13.82188765312757 Gly with method 3

In [9]:
R_Edd/(constants.light_year*10**9)

13.82188765312757

=========================================================================================================================

# Francis M. Sanchez Universe Radius calculus in 3 minutes (method 4)
$$R_{Francis.M.Sanchez}=ƛ_e.g(6)$$

With $$g(k)=exp(2^{k+1/2})/k$$ when $$k=6$$

And $$ƛ_e=\frac{\hbar}{m_e.c}$$ as the reduced Compton electron wavelength radius

In [10]:
import numpy as np
from scipy import constants
R_FMS=(constants.hbar/(constants.m_e*constants.c))*np.exp(2**(6+(1/2)))/6
print(R_FMS)

1.307568892158173e+26


## One obtains 13.821014095558315 Gly with method 4

In [11]:
R_FMS/(constants.light_year*10**9)

13.821014095558315

=========================================================================================================================

# Universe Radius calculus based on Rydberg reduced wavelength (method 5) 
$$R_{Rydberg}=\frac{(ƛ_{Ryd}.n^4)^2}{ƛ_p}$$

In [12]:
a=constants.fine_structure**-1
print(a)

137.0359990836958


In [13]:
a**2

18778.865044866678

In [14]:
lambdabarRyd=2*(constants.hbar/(constants.m_p*constants.c))*(a**2)
print(lambdabarRyd)

7.898725289848868e-12


In [50]:
R_Ryd=(lambdabarRyd**2)/(constants.hbar/(constants.m_p*constants.c))
print(R_Ryd)

2.966581924890947e-07


In [51]:
# Incorrect result , method to be modified
R_Ryd/(constants.light_year*10**9)

3.13567956881208e-32

=========================================================================================================================

# Universe Radius calculus based on pi (method 6) 
$$R_{\pi}=ƛ_e.\pi^{155/2}$$

In [17]:
R_pi=(constants.hbar/(constants.m_e*constants.c))*constants.pi**(155/2)
print(R_pi)

1.3058148293550039e+26


## One obtains 13.802473637099501 Gly with method 6

In [18]:
R_pi/(constants.light_year*10**9)

13.802473637099501

=========================================================================================================================

# Universe Radius calculus based on 2 (method 7) 
$$R_{2}=ƛ_e.2^{128}$$

In [19]:
R_2=(constants.hbar/(constants.m_e*constants.c))*2**128
print(R_2)

1.3140318971018876e+26


## One obtains 13.889328111716429 Gly with method 7

In [20]:
R_2/(constants.light_year*10**9)

13.889328111716429

=========================================================================================================================

# Universe Radius calculus based on Fermion and Kotov period (method 8) 
$$R_{kotov-fermi}=(2.(c.t_k)^2)/(F^2.ƛ_e)$$

In [22]:
R_kf1=(2*(constants.c*9600.6))/(573007.82**2*(constants.hbar/(constants.m_e*constants.c)))
print(R_kf1)

45400591780805.734


$$R_{kotov-fermi}=\frac{((2.t_k)^2/ƛ_e)}{F^2}$$

In [23]:
R_kf=(2*(constants.c*9600.6)**2)/(constants.hbar/(constants.m_e*constants.c)) / 573007.82**2
print(R_kf)

1.3067141449737733e+26


https://en.wikipedia.org/wiki/Light-year

In [24]:
# https://en.wikipedia.org/wiki/Light-year
constants.light_year

9460730472580800.0

In [25]:
R_kf/(constants.light_year*10**9)

13.811979410690407

=========================================================================================================================

# Universe Radius calculus based on "a"  (method 9) 
$$a=1/\alpha$$ $\alpha$ being the fine structure constant  $$R_{a}=ƛ_e.(2.\pi^2.a^3)^5$$

In [26]:
lambdabare=(constants.hbar/(constants.m_e*constants.c))
print(lambdabare)

3.8615926796089057e-13


In [27]:
(2*constants.pi**2)*((1/constants.fine_structure)**3)**5

2.2276645699260076e+33

In [28]:
R_a=((constants.pi**2*(2*(1/constants.fine_structure)**3))**5)*(constants.hbar/(constants.m_e*constants.c))
print(R_a)

1.3059760854303987e+26


In [29]:
R_a/(constants.light_year*10**9)

13.80417811516134

=========================================================================================================================

# Universe Radius calculus based on $c$ and $$M_U$$ (method 10) 


$$R_U = \frac{2\cdot G\cdot M_U}{c^2}$$

$$\frac{m_p^4}{m_p\cdot m_H\cdot\ m_e} = M_U$$

In [37]:
# Use m_H instead of m_n
M_U=((constants.m_p)**4)/(constants.m_p*constants.m_e*constants.m_n)
print(M_U)

3.066961657680341e-24


In [47]:
R_method10=(2*constants.G*M_U)/constants.c**2
print(R_method10)

4.555149761835214e-51


In [45]:
R_method10=2*constants.G/constants.c**2
print(R_method10)

1.4852320538237331e-27


In [48]:
# Incorrect result , method to be modified
R_method10/(constants.light_year*10**9)

4.81479709736685e-76

=========================================================================================================================

# Measurement Confirmation of the methods given above by:

### Confirmation Measurement #1
Universe Radius measurement based on supernovae 1a
$$R_{S1a}=13.6(6)Gly$$

### Confirmation Measurement #2 (Planck Space Observatory)
https://en.wikipedia.org/wiki/Planck_(spacecraft)
13.8 billion years (as of 2015) – 13.799±0.021 billion (109) years within the Lambda-CDM concordance model.

$$R_{Planck}=13.799±0.021 Gly$$

### Confirmation Measurement #3 (James Webb Space Telescope)

#### https://en.wikipedia.org/wiki/James_Webb_Space_Telescope

Measurements due to start in 2021

=========================================================================================================================

# Other estimates

## Universe mass calculus given by the relation:

In [5]:
%%latex
\begin{align}
\frac{m_p^4}{m_p\cdot m_H\cdot\ m_e} = M \\
\end{align}

<IPython.core.display.Latex object>

In the following example, one can write the equation with Latex:

In [6]:
from IPython.display import Latex
Latex(r"""\begin{eqnarray}
\frac{m_p^4}{m_p\cdot m_H\cdot\ m_e} = M  \\
\ R_U = \frac{2\cdot G\cdot M_U}{c^2} \\
\ M_U = \frac{({ħc}{G})^2}{m_em_pm_h} \\
\frac{r_H}{ƛ_e} = \frac{\sum\frac{1}{n}}{\sum\frac{1}{n^2}}\\
\end{eqnarray}""")


<IPython.core.display.Latex object>

=========================================================================================================================

:star:

# Docs

This project aims at providing a repository dedicated to Dr Francis Michel Sanchez's work in holography and cosmology; it is just a mere continuation of what was initiated and published in a set of theoritical physics articles.


## Publications
* Please, cite this paper (http://vixra.org/pdf/1811.0146v8.pdf) if you employ this package in your research work.
* [Vixra Publications](http://vixra.org/author/francis_m_sanchez)
* [Rxiv Publications](http://rxiv.org/author/f_m_sanchez)
* [A coherent resonant cosmology approach and its implications in microphysics and Biophysics Pages 375-407 of Quantum Systems in Physics, Chemistry](https://www.springer.com/fr/book/9783319502540?token=gbgen&wt_mc=GoogleBooks.GoogleBooks.3.EN#)
* [From Coherent Cosmic Oscillations to the Steady-State Cyclic Ultrafast](https://rxiv.org/abs/1401.0228)
* [Evidence for a Steady-State, Holographic, Tachyonic and Super-Symmetric Cosmology](https://www.researchgate.net/publication/267430152_Evidence_for_a_Steady-State_Holographic_Tachyonic_and_Super-Symmetric_Cosmology)
* [academia.edu Correcting Five Historical Faults in Modern Cosmology](https://www.academia.edu/30843365/Correcting_Five_Historical_Faults_in_Modern_Cosmology)

## Videos

* [L'enigme des grands nombres Part1](https://www.youtube.com/watch?v=ev_eIqLtK8s)
* [L'enigme des grands nombres Part2](https://www.youtube.com/watch?v=i4w392nYqns)
* [L'enigme des grands nombres Part3](https://www.youtube.com/watch?v=b4WvI0yVXnE)
* [Exposés de Francis Sanchez playlist created by J.Bailhache](https://www.youtube.com/playlist?list=PLQXUNrzoAeFbmfHo3jgWzzExlVoBn2JUw)
* [Khelif/Torgoz/Sanchez' Debate on LE FORMALISME EN PHYSIQUE ET EN SCIENCE](https://www.youtube.com/watch?v=TEK2hCI2we0)

* [La science du 3eme millenaire playlist created by J.Bailhache](https://www.youtube.com/playlist?list=PLQXUNrzoAeFbLOuEsltAUWJWCcAEgeGJE)
* [Francis Sanchez Cosmologie 23/03/2017](https://www.youtube.com/watch?v=NmvvIQjxBw8)
* [La complexité mathématique face à la simplicité cosmique - Exposé de Francis Sanchez](https://www.youtube.com/watch?v=dyUlUahgHGg)
* [La méthode scientifique en question et les constantes fondamentales - Exposé](https://www.youtube.com/watch?v=zUt1hxTdiuA)
* [Jean Maruani chez lui et Francis Sanchez autour du nombre 237 29. 12. 17](https://www.youtube.com/watch?v=05g416VWjnw)
* [Pr. Michel Grosmann: Holographie quanto-cosmique](https://www.youtube.com/watch?v=M7bJnekDIt0)
* [Pr. Renée Veysseyre: La constante d'Atiyah réunifie Physique et Mathématique](https://www.youtube.com/watch?v=mbPpKqakDAU)
* [Christian Bizouard, astronome: Omniprésence astrophysique du cycle de Kotov](https://www.youtube.com/watch?v=DGeBC73kKvQ)
* [Pr. Francis Sanchez: Historique, Principe Calculatoire, Axe Topologique et Principe Holique](https://www.youtube.com/watch?v=IgKSrfsqFEs)

## Books

* [Bizouard, Kotov, Sanchez wrote in 20th Natural Philosophy Alliance Proceedings by David de Hilster, published by Lulu.com in 2013]()
* [Quantum Systems in Physics, Chemistry, Springer Ed](https://www.amazon.com/gp/search?index=books&linkCode=qs&keywords=9783319502557)
* [The Worldwide List of Alternative Theories and Critics](https://books.google.com/books?isbn=2902425171)
* [The Dirac Electron: From Quantum Chemistry to Holistic Cosmology by J.Maruani](https://onlinelibrary.wiley.com/doi/abs/10.1002/jccs.201500374#)
* [solar 22 years cycle](https://link.springer.com/article/10.1007%2Fs10509-016-2985-8)
* [Current Issues in Cosmology](https://www.cambridge.org/gb/academic/subjects/physics/cosmology-relativity-and-gravitation/current-issues-cosmology?format=HB&isbn=9780521858984)
* ```@inproceedings{Sanchez2015RemarkablePO,
  title=Remarkable Properties of the Eddington Number 137 and Electric Parameter 137 . 036 excluding the Multiverse Hypothesis},
  author={Francis M. Sanchez and Valery A. Kotov,
  year=2015```

## Papers

* [College de France (2004): Discussions sur les oscillations cosmiques, les nombres sans dimension et les periodicites en microphysique et cosmologie - College de France 27th Feb 2004](https://lempel.pagesperso-orange.fr/occ_cdf.pdf)

More to come soon stay tuned.../.:stars:.

=========================================================================================================================

In [9]:
#Write your equation in a dedicated cell with latex

from IPython.display import Latex
Latex(r"""\begin{eqnarray}
\frac{m_p^4}{m_p\cdot m_H\cdot\ m_e} = M \\
\frac{\pi_Pt}{\pi_Bb} = 6 \cdot \frac{\pi^5}{p_H^9} \\
\end{eqnarray}""")

<IPython.core.display.Latex object>

In [23]:
#Write Bekenstein-Hawking Entropy 
from IPython.display import Latex
Latex(r"""\begin{eqnarray}
S = \frac{c^3 k A}{4 \hbar G} \\
\end{eqnarray}""")

<IPython.core.display.Latex object>